compute_tools/src/compute.rs

```diff
@@ -1540,6 +1563,41 @@ impl ComputeNode {
             });
         }
 
+        // If extended compute audit is enabled configure and start rsyslog
+        // We check that the audit_log_level changed compared to the previous spec and skip this step if not.
+        let current_audit_log_level = self.get_current_audit_log_level();
+
+        match spec.audit_log_level {
```

Comment:
This code looks like a repeat of the previous code. Can you refactor it into a function?

Reply:
It is very similar, but logic is different, because one is called from start_compute() and other from reconfigure().
I refactored the bit with env variable, but prefer to keep other parts visible in the calling functions. WDYT?